utilities/src/main/java/io/syndesis/qe/Component.java

```diff
@@ -69,9 +69,11 @@
      * @return list of pods of syndesis components
      */
     public static List<Pod> getComponentPods() {
-        return OpenShiftUtils.getInstance().pods().withLabel("syndesis.io/component").list().getItems().stream()
+        List<Pod> syndesisComponents = OpenShiftUtils.getInstance().pods().withLabel("syndesis.io/component").list().getItems().stream()
```

Comment:
I think we can use `Component.getAllComponents()` and then just select all pods starting with syndesis-<component name>, that will also select syndesis-jaeger when jaeger is enabled

Reply:
selecting `syndesis-jaeger` is not a problem, it contains `syndesis.io/component` label.

Reply:
That will not work with todo app